src/figma-notification.ts

```diff
@@ -28,11 +28,11 @@ const isEvent = (req: express.Request) => {
 /*
 Figma Webhookでは時間差でretriesが0の複数のリクエストを送ってくることあるため、
 イベント発生時間とタイムスタンプの時間差を元に処理の継続を判断する。
-ここでの基準は二つのパラメータの値の差が5秒以下。
+ここでの基準は二つのパラメータの値の差が30秒以下。
 */
 const isFirstRequest = (event) => {
   if (event.retries === 0) {
-    return new Date(event.timestamp).getTime() - new Date(event.created_at).getTime() < 5000;
+    return new Date(event.timestamp).getTime() - new Date(event.created_at).getTime() < 30 * 1000;
```

Comment:
他に `* 1000`の書き方しているところがなければ、ここは`30000`で良いと思います。
気持ちはわかるけど、わざわざ`* 1000`にしなくても大丈夫。
```suggestion
    return new Date(event.timestamp).getTime() - new Date(event.created_at).getTime() < 30000;
```

Reply:
@y047aka 
横から失礼します。
個人的に長い数字をパッと認識できなくて `* 1000` をよく書いてしまうので今後の参考にしたいのですが、一般的にはmsをそのままの値で書いた方が良いのでしょうか？

Reply:
ここだけで言えば、もとの5秒が`5 *  1000`ではなかったので、30秒も `30000`で良いという考えでした。
特にルールを定めてるわけではないので、重箱の隅をついてしまった感はあるかも。

改めてsiiibo-appのコードで事例を探してみると、「分」を示す目的では`* 60 * 1000`がありました。
「秒」の表現は「分」に比べれば簡単ですが、ミスする可能性を減らす目的で明示的に`* 1000`を使っても良いと思います。

私自身、他の人がどのように判断するか気になります。

Reply:
なるほど、ありがとうございます！

Reply:
以下コピペ

-----

Elixirのようにコンパイル時処理があって、`@value`のようにコンパイル時計算されるmodule attributeとかなら掛け算で書いても実行時オーバーヘッドはないので、掛け算で書いてスケールをわかりやすくするのはあり。
ただElixirの場合、単に桁数を明示したいだけなら`1_000_000`記法があるのでこの目的では掛け算表記にする必要はなくて、時間に係る数値（60進数）など10進計算すると意図がわかりにくい数字わかりやすくするために`7 * 24 * 60 * 60`のような書き方をするくらい。
桁区切り記法がないElmなどは可読性を求めるなら掛け算表記くらいしかないけど、実行時オーバーヘッドになる。とはいえ現代のJSエンジンを信じていいんじゃないかな。定数ならJITコンパイラがインライン化してくれたりするんじゃないだろうか…Elmがコンパイル生成する掛け算表記の数値ってどんなんだったかな
興味があったらこういうのどこまで最適化されるかは調べてみるといいと思う。
ちなみにJSにも桁区切り記法はあって最近の殆どのブラウザとnode 12.5以降ならもう使える。リンクしたPRのところならこれ使うといいんじゃないか。GASじゃなくてHerokuで動かしてるコードなのでnodeだし。GASで対応してるかはわからぬ
https://github.com/tc39/proposal-numeric-separator
https://github.com/nodejs/node/issues/28427